subcommand/server-acl-init/command.go

```diff
@@ -115,6 +118,10 @@ func (c *Command) Run(args []string) int {
 		c.UI.Error(fmt.Sprintf("%q is not a valid timeout: %s", c.flagTimeout, err))
 		return 1
 	}
+	if c.flagReleaseName == "" {
+		c.UI.Error("-release-name must be set")
```

Comment:
If this isn't set, we won't find the Consul servers since we do a lookup by label so best to give an error.